packages/core/useFetch/index.test.ts

```diff
@@ -8,17 +8,10 @@ describe('useFetch', () => {
   }
 
   test('basic get request', async() => {
```

Comment:
Looks like jest doesn't support fetch, so it seems like we need to mock it. There are a few libraries online that provide fetch mocks, like this one https://www.npmjs.com/package/jest-fetch-mock, but I wanted to get some input from you before I did anything

Reply:
Sure, go for it!